sql/seta_schema.sql

```diff
@@ -2,7 +2,7 @@ USE ouija;
 
 CREATE TABLE IF NOT EXISTS `seta` (
   `id` int(11) NOT NULL AUTO_INCREMENT,
-  `jobtype` varchar(64) NOT NULL,
+  `jobtype` varchar(1024) NOT NULL,
```

Comment:
I meet **_mysql_exceptions.DataError: (1406, "Data too long for column 'jobtype' at row 1")** problem. Maybe change it to 512 or 256 is more reasonable, but 64 is too small anyway. :)